### PR TITLE
Search view for video working with redirect

### PR DESCRIPTION
**Pull request by kobusLink**

### CODE REVIEW

File: VideoStream/FrontEnd/app.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "pages": [
+    "pages/seach-page/seach-page",
```
Comment (Stehan19) [inline VideoStream/FrontEnd/app.json:3]: shouldn't the index page be first?

Comment (kobusLink) [reply VideoStream/FrontEnd/app.json:3]: index page doesnt go anywhere atm, at least with this Hugo can test his stuff

Comment (kobusLink) [reply VideoStream/FrontEnd/app.json:3]: Can change to be more "clean"

Comment (130nk3r5) [reply VideoStream/FrontEnd/app.json:3]: We can put the index page back in place....
Has @nicferretti checked in yet?

**Pull request merged**